chasm/lib/buf.yaml

```diff
@@ -7,6 +7,7 @@ breaking:
     - WIRE
   ignore:
     - chasm/lib/scheduler/proto/v1/message.proto
+    - chasm/lib/callback/proto/v1/tasks.proto
```

Comment:
We should have a TODO to remove these after merging this PR. I'd be fine to leave the scheduler one though and let @lina-temporal handle that.